Jellyfin.Plugin.Anime/Constants.cs

```diff
@@ -0,0 +1,7 @@
+namespace Jellyfin.Plugin.Anime
+{
+    static class Constants
+    {
+        public const string UserAgent = "jellyfin-plugin-anime";
```

Comment:
I would propose we use a more proper user-agent string, such as;
`jellyfin-plugin-anime/5.0.0.0 (+https://github.com/jellyfin/jellyfin-plugin-anime/issues) Jellyfin/10.5.0`

https://en.wikipedia.org/wiki/User_agent has examples of how browsers and bots behave in this regard

Reply:
Let's save that for another pull request. This is an improvement and I'm sure people want it working again after the update so I'll push a new version for now.